apps/web/src/components/forms/profile.tsx

```diff
@@ -138,6 +138,7 @@ export const ProfileForm = ({ className, user }: ProfileFormProps) => {
                     containerClassName={cn('rounded-lg border bg-background')}
                     defaultValue={user.signature ?? undefined}
                     onChange={(v) => onChange(v ?? '')}
+                    allowTypedSignature={true}
```

Comment:
I hardcoded this since the typed signature should always be available on the profile page.

Reply:
👍🏻